facts/pi/gold_image.txt

```diff
@@ -3,8 +3,8 @@
 Instructions heavily influenced by https://youtu.be/T9AtKld8USU
 
 # Create bootable SDCARD:
-1. Download Raspbian Stretch Lite 
-  a. 2019-09-26-raspbian-buster-lite.img
+1. Download the current specified Raspbian image from https://www.raspberrypi.org/downloads/raspbian/
+   a. As of 2020-02-24 the specified image is 2020-02-13-raspbian-buster-lite.img
```

Comment:
@owendelong The change to line 6 makes sense to correct the `stretch` reference. However, which image did we actually use as our base on the pis for this year? I think that needs to be reflect for `a.` vs just the latest img as of writing this doc. So i believe that `a.` should still be: `2019-09-26-raspbian-buster-lite.img`